docs/pages/modules/native-module-tutorial.mdx

```diff
@@ -0,0 +1,459 @@
+---
+title: Creating your first native module
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+import { PlatformTag } from '~/ui/components/Tag';
+import { APIBox } from '~/components/plugins/APIBox';
+import { Terminal } from '~/ui/components/Snippet';
+
+In this tutorial we are going to build a module that stores the user's preferred app theme - either dark, light, or system. We'll use [`UserDefaults`](https://developer.apple.com/documentation/foundation/userdefaults) on iOS and [`SharedPreferences`](https://developer.android.com/reference/android/content/SharedPreferences) on Android. It is possible to implement web support using [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage), but we'll leave that as an exercise for the reader.
+
+## 1. Initialize a new module
+
+First, we'll create a new module. On this page we will use the name `expo-settings` / `ExpoSettings`. You can name it whatever you like, just adjust the instructions accordingly:
+
+<Terminal cmd={['$ npx create-expo-module expo-settings']} />
+
+> **Tip**: Since you aren't going to actually ship this library, you can hit <kbd>return</kbd> for all of the prompts to accept the default values.
```

Comment:
```suggestion
> **Tip**: Since you aren't going to actually ship this library, you can hit <kbd>↵ Return</kbd> for all of the prompts to accept the default values.
```

KBD tweak.